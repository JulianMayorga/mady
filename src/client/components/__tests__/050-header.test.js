/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../050-header';

jest.mock('react-dom');

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Header onShowSettings={() => {}} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
