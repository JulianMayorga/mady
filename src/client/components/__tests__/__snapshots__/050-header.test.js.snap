exports[`Header renders correctly 1`] = `
<div
  style={
    Object {
      "WebkitFlex": "0 0 2.5em",
      "alignItems": "center",
      "backgroundColor": "#dcd6ff",
      "display": "flex",
      "flex": "0 0 2.5em",
      "flexDirection": "row",
      "padding": "5px 8px"
    }
  }>
  <div
    style={
      Object {
        "WebkitFlex": "1",
        "flex": "1"
      }
    } />
  <div
    style={
      Object {
        "fontSize": "1.3em",
        "fontWeight": 900,
        "letterSpacing": 3
      }
    }>
    MADY
    <i
      className="giu-icon fa fa-cog"
      id="madyBtnSettings"
      onClick={[Function onShowSettings]}
      style={
        Object {
          "color": undefined,
          "cursor": "pointer",
          "letterSpacing": "normal",
          "marginLeft": 10
        }
      }
      title="Settings" />
  </div>
  <div
    style={
      Object {
        "WebkitFlex": "1",
        "flex": "1",
        "fontSize": "1.3em",
        "fontWeight": 900,
        "letterSpacing": 3,
        "marginRight": 10,
        "textAlign": "right"
      }
    }>
    <i
      className="giu-icon fa fa-question-circle"
      onClick={[Function onClick]}
      style={
        Object {
          "color": undefined,
          "cursor": "pointer",
          "letterSpacing": "normal"
        }
      } />
  </div>
</div>
`;
